src/macros.rs

```diff
@@ -39,3 +39,36 @@ macro_rules! lzma_debug {
 macro_rules! lzma_info {
     ($($arg:tt)+) => {};
 }
+
+// Macro to repeat elements of N times in an array without Clone
```

Comment:
This mentions `Clone`, but `BitTree` is already `Clone`.
I'd rather not import a macro from a random StackOverflow question (what is the license?).

Alternatively, copy pasting the `BitTree::new` N times in the source code is fine for me.

Reply:
brainfart, I meant `Copy`, which `BitTree` is not. I've just copy-pasted `BitTree::new` N times instead if you're fine with that.